### DISCUSSION TITLE
In-place resource search

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#1130] [Feature requests]
Following [this](https://discord.com/channels/740892036978442260/740893011994738751/1006583814245601300) discussion, there might be an improvement in doing in-place search for resource search instead of opening a pop-up.

**Comment by bear-in-mind** [comment]
Just encountered the need for that.

What about keeping the pop up as is, and adding a submit button ? That way we can still use the pop up for quick results, and either hit `ENTER` or click the `Submit` button to get the page with the searched resource.

**Reply by adrianthedev** [reply]
I think it's easier to just support the in-place version. Less context switching and more information for the user.

The only issue is that we don't have the time allocation for this in the next few months. The resource search works well how it is, and this would be just an improvement. We're now focused on bringing more powerful features to Avo and solving more painful issues for developers.

We're open for others to come in and PR this functionality ✌️

**Reply by bear-in-mind** [reply]
Oh sure, I'd be glad to take this on :) I need it for myself anyway !

**Reply by adrianthedev** [reply]
That would be amazing! I appreciate it! Let me know if you want to have a chat or anything to help you navigate the code or see about an approach.

**Comment by Paul-Bob** [comment]
<img width="2121" height="432" alt="image" src="https://github.com/user-attachments/assets/243d547e-4fb7-4a82-a65f-d32f6b946209" />
<img width="2123" height="636" alt="image" src="https://github.com/user-attachments/assets/ef00c42a-c5ec-4a84-b797-85481494bc05" />

Implemented on Avo 4
